src/amo/components/Routes/index.js

```diff
@@ -162,6 +163,12 @@ const Routes = ({ _config = config }: Props = {}): React.Node => (
       component={TagPage}
     />
 
+    <Route
+      exact
+      path="/:lang/:application(firefox|android)/feedback/addon/:addonIdentifier/"
```

Comment:
nit (1): `addonIdentifier` -> `addonId`? or just `id`

nit (2): why `/feedback/addon/:id/` and not `/addon/:id/feedback/` like we have for "all versions", statistics, etc.?

Reply:
`addonId` I thought could be confused with then numeric add-on id.  The url structure was because there is going to be a `/feedback/user/`, a `/feedback/collection/` and a `/feedback/rating/` too - and the expected id is a guid, whereas the `id` in the canonical `/addon/:id/xxx` is an add-on slug.

Reply:
Ha, I see. Then I guess we keep `/feedback/` as a prefix in the path.

As for the `id` in the add-on detail page, it accepts: slug, addon numeric id, guid. That isn't what this feedback page does at the moment, but the add-on isn't loaded in this page so maybe that's fine? If we only want the `guid`, then maybe we should have `:guid` in the path?